nmdc_runtime/site/ops.py

```diff
@@ -274,6 +287,41 @@ def create_objects_from_ops(context, op_docs: list):
     return op_docs
 
 
+@op(required_resource_keys={"runtime_api_user_client"})
+def validate_metadata(context: OpExecutionContext, database: nmdc.Database):
+    client: RuntimeApiUserClient = context.resources.runtime_api_user_client
+    response = client.validate_metadata(database)
+    response.raise_for_status()
+    body = response.json()
+    if body["result"] != "All Okay!":
+        raise Failure(
+            description="Metadata did not validate",
+            metadata={"detail": body["detail"]},
+        )
+    return body
+
+
+@op(required_resource_keys={"runtime_api_user_client"})
+def submit_metadata_to_db(context: OpExecutionContext, database: nmdc.Database) -> str:
+    client: RuntimeApiUserClient = context.resources.runtime_api_user_client
+    response = client.submit_metadata(database)
+    response.raise_for_status()
+    body = response.json()
+    return body["detail"]["run_id"]
+
+
+@op(required_resource_keys={"runtime_api_user_client"})
+def poll_for_run_completion(context: OpExecutionContext, run_id: str) -> RunSummary:
+    client: RuntimeApiUserClient = context.resources.runtime_api_user_client
+    response = client.get_run_info(run_id)
+    response.raise_for_status()
```

Comment:
`response.raise_for_status()` is redundant now, as I added it to `RuntimeApiUserClient.request`. 🙂 Please remove here.